app/mixins/custom-form.js

```diff
@@ -196,7 +196,7 @@ export default Mixin.create(MutableArray, {
       this.store.createRecord('custom-form', {
         fieldIdentifier : 'sponsorshipRequired',
         form            : 'speaker',
-        type            : 'text',
+        type            : 'select',
```

Comment:
@kushthedude Won't it break all the past events?

Reply:
A migration will be needed on server side once the PR is merged